rush.json

```diff
@@ -527,6 +527,14 @@
       "versionPolicyName": "alpha"
     },
 
+    {
+      "packageName": "@kadena/react-ui",
+      "projectFolder": "packages/libs/react-ui",
+      "tags": ["libs"],
+      "shouldPublish": false,
+      "versionPolicyName": "alpha"
```

Comment:
Don't add a `versionPolicyName` if you don't want it to be published. Unintuitively the `rush publish --include-all` option includes ones with `versionPolicyName`

```bash
$ rush publish --help 
 --include-all         If this flag is specified, all packages with 
                        shouldPublish=true in rush.json or with a specified 
                        version policy will be published if their version is 
                        newer than published version.
```